pocs/tests/test_observatory.py

```diff
@@ -69,20 +87,44 @@ def test_bad_mount_driver(config):
         Observatory(simulator=simulator, config=conf, ignore_local_config=True)
 
 
-def test_bad_scheduler(config):
+def test_can_observe_no_scheduler(config):
     conf = config.copy()
-    simulator = ['all']
-    conf['scheduler']['type'] = 'foobar'
-    with pytest.raises(error.NotFound):
-        Observatory(simulator=simulator, config=conf, ignore_local_config=True)
+    cameras = create_cameras_from_config(conf)
+
+    obs = Observatory(cameras=cameras,
+                      scheduler=None,
+                      config=conf
+                      )
 
+    assert obs.can_observe is False
 
-def test_bad_scheduler_fields_file(config):
+
+def test_can_observe_no_cameras(config):
     conf = config.copy()
-    simulator = ['all']
-    conf['scheduler']['fields_file'] = 'foobar'
-    with pytest.raises(error.NotFound):
-        Observatory(simulator=simulator, config=conf, ignore_local_config=True)
+    details = setup_site_location_details_from_config(config)
+    scheduler = create_scheduler_from_config(config, observer=details['observer'])
+
+    obs = Observatory(cameras=None,
+                      scheduler=scheduler,
+                      config=conf
+                      )
+
+    assert obs.can_observe is False
+
+
+def test_can_observe_no_mount(config):
+    conf = config.copy()
```

Comment:
We haven't yet moved the mount creation out of the Observatory so probably didn't need to modify the `can_observe` at this point, which means we don't need to create this test right now.